### ISSUE TITLE
duplicate Id issue

### ISSUE THREAD

**Issue by akashmdkml** [dequelabs/axe-core#2291]
###  Duplicate id issue:
_As per the w3c standard the content wrapped inside area-hidden="true" will not be readable by screen readers, The axe mark the following code as duplicate Id but the google lighthouse does not mark this issue because the google will identify duplicate id by aria-labelledby, so the lighthouse finds duplicate Id's only which is readable by the screen. (https://web.dev/duplicate-id-aria/#how-to-fix-duplicate-ids)
Would like to know the community view on this issue, please provide your inputs, Thanks_
`<div id="duplicate-id">Test Duplicate ID</div>
        <div>some SVG
            <svg aria-hidden="true" viewBox="0 0 1000 1000" xmlns="http://www.w3.org/2000/svg" stroke="red" fill="grey">
            <circle id="duplicate-id" cx="50" cy="50" r="40" />
            </svg>
        </div>`

Expectation: << I think it should not mark as an issue for duplicate id's which are wrapped inside aria-hidden=true >>

Actual: << marking as an issue for duplicate id's which are wrapped inside aria-hidden=true >>

Motivation: << we use SVG icons site-wide, but all those elements are wrapped inside aria-hidden, so I think as per WCAG standard it should ignore the element wrapped inside the aria-hidden=true>>

**Comment by WilcoFiers** [comment]
@akashmdkml This is intentional. IDs do not get ignored if an element is hidden. For instance if you use document.getElementById(), you'll get a hidden element back. This works the same in accessibility. If you reference an element with aria-labelledby for example, even if the element you reference is hidden, that element will still be used to create the label. Hidden elements with duplicate IDs can, and do create accessibility issues.

That said, if the element isn't referenced with ARIA, and the element isn't focusable, it is unlikely to cause issues. That scenario axe-core reports as a minor issue. The HTML specifications explicitly say not to use duplicate IDs, regardless of if they are hidden. This is what the WCAG success criterion 4.1.1 is based on. Consensus at the WCAG working group is that this is a failure of WCAG, albeit a minor one.

Hope that helps clarify. Let me know if you have any other questions.
